cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -307,6 +316,17 @@ fn zkasm_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut OperandC
             collector.reg_clobbers(clobbered);
             collector.reg_fixed_def(rd, c0());
         }
+        &Inst::RemArith32 { rd, rs1, rs2, .. } => {
```

Comment:
As this and the branch above do not differ, merge them:

```rust
Inst::RemArith { rd, rs1, rs2, .. } | Inst::RemArith32 { rd, rs1, rs2, .. } => { ... }
```

Reply:
Hm, so forward looking I was thinking that we’d perhaps do something to abstract away some common parts about these different instructions to reduce duplication, but I guess you’re right that it makes no sense to do so especially if we’re looking at these code paths diverging even further as development proceeds.